### ISSUE TITLE
Import external css files

### ISSUE THREAD

**Issue by foreggs** [ionic-team/stencil#105]
I'd like to import a css file alongside the component, is this possible like so or using a different syntax?

This is what I use with react, but it fails in stencil, can you add a css-loader or provide us with an alternative solution? Thanks 🔢 
```js
import "@material/button/dist/mdc.button.css";
```

**Comment by foreggs** [comment]
Using css imports works, but is there any de-facto stencil standard?
```css
@import "@material/button/dist/mdc.button.css";
```

**Comment by adamdbradley** [comment]
You can use `styleUrls` to import external css files. Importing CSS through ES module imports is not something on our plan to support.

**Comment by cipriancirstea** [comment]
How would one use `styleUrls` to import multiple css files?

**Comment by rtpHarry** [comment]
@cipriancirstea I guess you solved this or gave up, but I just hit this issue and solved it by making it an array, like:

```
@Component({
  tag: 'pharm-modal-dialog',
  styleUrls: [
    '../../../../node_modules/simplebar/dist/simplebar.min.css',
    'modal-dialog.css',
  ],
  shadow: false
})
```

I don't know if there is a better way to do this with a less brittle path, but trying to put the path starting with `@simplebar/...` didn't work.

**Comment by milobluebell** [comment]
only first one will be applied

**Comment by will-hu-0** [comment]
Try this, at least it works for me.

```
@Component({
  tag: 'your-component,
  styleUrl: 'your-component.scss',
  shadow: true,
})
export class YouComponent {
...
render() {
  return (
    <Host>
      <link rel="stylesheet" href="your CSS" />
      <div......>
    </Host>
  )
}
```

**Comment by christian-bromann** [comment]
We recently released a fix that resolved the bug around `styleUrls`. A Stencil component should now be able to read from multiple urls, e.g.:

```ts
@Component({
  tag: 'my-component',
  styleUrls: ['reset.scss', 'roboto.scss', 'css-variables.scss', 'core.scss'] 
})
export class MyComponent {
  // ...
}
```

see https://github.com/ionic-team/stencil/pull/5131. It was released with `v4.8.2`. Please let me know if there are still issues with this.

**Comment by gregorypratt** [comment]
@will-hu-0 won't the browser make multiple network requests to that stylesheet in the `link` tag if it is referenced from multiple components?

**Comment by will-hu-0** [comment]
> @will-hu-0 won't the browser make multiple network requests to that stylesheet in the `link` tag if it is referenced from multiple components?

It does make multiple requests.. 
I have tested with the solution provided by @christian-bromann , and it works like a charm:


> We recently released a fix that resolved the bug around `styleUrls`. A Stencil component should now be able to read from multiple urls, e.g.:
> 
> ```ts
> @Component({
>   tag: 'my-component',
>   styleUrls: ['reset.scss', 'roboto.scss', 'css-variables.scss', 'core.scss'] 
> })
> export class MyComponent {
>   // ...
> }
> ```
> 
> see #5131. It was released with `v4.8.2`. Please let me know if there are still issues with this.
